chainer/link_hooks/spectral_normalization.py

```diff
@@ -1,9 +1,14 @@
+import numpy
+
+import chainer
 from chainer import backend
 from chainer import configuration
 import chainer.functions as F
 from chainer import link_hook
 import chainer.links as L
 from chainer import variable
+import chainerx
+from chainerx._fallback_workarounds import _from_chx as from_chx_workaround
```

Comment:
uh-oh...
Do I have to fix this as follows?
```python
import chainerx._fallback_workarounds as fallback

...

xp, device, array = fallback._from_chx(some_array)
```

Reply:
Yes, please.